torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -15,7 +15,13 @@ static const std::unordered_map<Symbol, Symbol> alias_map = {
     {aten::linalg_det, aten::det},
     {aten::outer, aten::ger},
     {aten::arccosh, aten::acosh},
-    {aten::arccosh_, aten::acosh_}};
+    {aten::arccosh_, aten::acosh_},
+    {aten::arccos, aten::acos_},
```

Comment:
shouldn't this be the non-inplace variant?

Reply:
Yep. Good catch.

Reply:
Also: let's elaborate the test to catch errors like this just in case they occur again.